### ISSUE TITLE
Slayer plugin not functioning at all

### ISSUE THREAD

**Issue by greyballs** [runelite/runelite#3281]
Everything is ticked on in the plugin. Neither the infobox or highlighting work. In the case of the infobox I have checked gem to no avail. I have tried reinstalling. I have tried everything.

**Comment by deathbeam** [comment]
Are you sure the root option (the slider) is turned on?

**Comment by greyballs** [comment]
yes

**Comment by SomeZer0** [comment]
Counter shows on the Gem, but there's no infobox. So the plugin is on.
![Additional Info](https://i.imgur.com/9ggtq96.png)

**Comment by greyballs** [comment]
ahhhhhhhhhhhhhhh so i need helm or ring for it to work?

**Comment by SomeZer0** [comment]
>ahhhhhhhhhhhhhhh so i need helm or ring for it to work?
I don't think so? I was just providing more context for the issue you were having. Not stating that the behavior is intended.

**Comment by greyballs** [comment]
oh ok. that still wouldnt explain why highlight doesn't work either

**Comment by Ash951** [comment]
It is not working for me either, but was pre-update.

**Comment by Kamielvf** [comment]
The client doesn't know what task you have unless you either:
- Receive a new task
- Check your slayer helm/ring

If one of these actions is performed, it should show the infobox correctly + the overlay on helm/ring.
The number on ring the user greyballs was talking about is most likely the teleport charges.

**Comment by Ash951** [comment]
awesome, all working yeah. thanks

**Comment by greyballs** [comment]
well highlighting as well as the info box is not working for me. I highly doubt getting a new task will fix it considering how many times I've tried checking my gem.

**Comment by Rowan-Mulder** [comment]
I had some issues with the **!task** command from the internal Slayer plugin.
Tried to find this configuration manually by looking at settings.properties, but after resetting the plugin...
![image](https://github.com/runelite/runelite/assets/62886548/8d1cb94d-520d-4d09-8a5d-410b165e10ac)
It's working now and I figured it's stored in the profiles2 directory now.
I hope this may help others experiencing the same issue.
